Mizu/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_Mizu.png DESTINATION reactos/Web/Wallpaper FOR all)
```

Comment:
Nice idea to keep these so that one can plug the reactos/wallpapers into our source tree and get the thing ready to compile.